From 98ecd5d9edcfc346aa506831062319e805bdfa80 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Mon, 3 Dec 2018 14:21:44 +0000
Subject: [PATCH] CMakeLists.txt: use find_package to find glog instead of
 pkg-config

* the 0.3.5 version in meta-oe is now using cmake instead of autotools for
  build and unlike 0.3.4 version doesn't provinde pkg-config file anymore

OE qemux86@luneos ~/build/wpb/webos-ports/tmp-glibc/work/i586-webos-linux/glog $ diff -rq 0.3.*/image
Only in 0.3.5-r0/image/usr/include/glog: config.h
Files 0.3.4-r0/image/usr/include/glog/logging.h and 0.3.5-r0/image/usr/include/glog/logging.h differ
Files 0.3.4-r0/image/usr/include/glog/raw_logging.h and 0.3.5-r0/image/usr/include/glog/raw_logging.h differ
Only in 0.3.5-r0/image/usr/lib: cmake
Only in 0.3.4-r0/image/usr/lib: libglog.a
Files 0.3.4-r0/image/usr/lib/libglog.so and 0.3.5-r0/image/usr/lib/libglog.so differ
Files 0.3.4-r0/image/usr/lib/libglog.so.0 and 0.3.5-r0/image/usr/lib/libglog.so.0 differ
Only in 0.3.4-r0/image/usr/lib: libglog.so.0.0.0
Only in 0.3.5-r0/image/usr/lib: libglog.so.0.3.5
Only in 0.3.4-r0/image/usr/lib: pkgconfig
Only in 0.3.4-r0/image/usr: share


Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index adc2bac..0f68e30 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,7 +14,7 @@ set(MTP_VERSION_MINOR 0)
 set(MTP_VERSION_PATCH 0)
 
 find_package(Boost REQUIRED COMPONENTS thread system filesystem unit_test_framework)
-pkg_check_modules(GLOG REQUIRED libglog)
+find_package(GLOG REQUIRED glog)
 
 set(
     MTP_HEADERS
